fog/view/server/src/fog_view_service.rs

```diff
@@ -119,6 +119,36 @@ where
         }
     }
 
+    fn auth_nonce_impl(
```

Comment:
:question: I may be missing the full migration picture, but `auth_impl` and `auth_nonce_impl` seem like pretty generic names for `client` vs `frontend` connections.
Are these going to be consolidated somehow as part of the migration or will these come out in the wash more clearly as connecting to one end and another connecting to another end?

Reply:
Changed accordingly.